fhir-audit/src/main/java/com/ibm/fhir/audit/cadf/model/CadfReason.java

```diff
@@ -138,6 +138,9 @@ public static String generate(CadfReason obj)
 
         public static void generate(CadfReason obj, JsonGenerator generator)
```

Comment:
maybe some javadoc to explain the behavior when null is passed?

Reply:
added comments